packages/back-end/src/routers/events/events.controller.ts

```diff
@@ -17,13 +18,13 @@ export const getEvents = async (
 ) => {
   const context = getContextFromReq(req);
 
-  if (!context.permissions.canViewEvents()) {
-    context.permissions.throwPermissionError();
-  }
-
   const events = await Event.getLatestEventsForOrganization(context.org.id, 50);
 
-  return res.json({ events });
+  return res.json({
+    events: events.filter((event) =>
+      context.permissions.canViewEvent(event.data as NotificationEvent)
+    ),
+  });
```

Comment:
Now that this endpoint is filtering the events based on the requester's permission level, it feels like we could loosen the permissions needed to view `app.growthbook.io/events`.

I could imagine if a user is in the habit of clicking on a Slack notification to view an event, they might find value in being able to view all events (that they have permission to view).

Reply:
If we do go this route - we will want to conditionally render the export button as we can keep that to admins only.

Reply:
I can definitely see the value in seeing a filtered event log for just the things you have access to / care about.  Might want to think some more about the best UX for that.  I'm not sure sending people to `Settings -> Logs` is the right solution.